{{cookiecutter.project_name}}/docs/template_usage.md

```diff
@@ -150,10 +154,20 @@ The following pre-commit hooks are for code style and format:
     black on Python code in docs.
 -   [prettier](https://prettier.io/docs/en/index.html):
     standard code formatter for non-Python files (e.g. YAML).
+-   [ruff][] based checks:
+    -   [isort](https://beta.ruff.rs/docs/rules/#isort-i) (`I`):
```

Comment:
When read from top to bottom, a user has no idea what the `I` means here. There should be some kind of pointer before this line.

Reply:
I've added some context, but maybe we should flip the order of "configuration" and "default checks" sections?

Reply:
Looks good, only the terminology is a bit off, see https://github.com/scverse/cookiecutter-scverse/pull/151#discussion_r1115420970